src/libcore/num/dec2flt/algorithm.rs

```diff
@@ -336,7 +336,7 @@ pub fn algorithm_m<T: RawFloat>(f: &Big, e: i16) -> T {
     round_by_remainder(v, rem, q, z)
 }
 
-/// Skip over most AlgorithmM iterations by checking the bit length.
+/// Skip over most Algorithm M iterations by checking the bit length.
```

Comment:
There is another `Algorithm M` in the file, I assumed it was a name for an algorithm that was implemented in the file...